### PR TITLE
Upgrade codebase to .NET6 and re-factor dependencies (1 of 3)

### PR DESCRIPTION
**Pull request by travisnielsen**
This PR replaces the C# Script (CSX) build process with a standard .NET6 project and removes dependencies on `rsvg` and `inkscape`. PNG files are now generated using [Playwright](https://github.com/microsoft/playwright-dotnet), which will result in much better consistency.

* Addresses #32. All icons captured in this issue are now render correctly. SVGs are converted to PNG via Playwright.
* Externalized config settings (file paths), which addresses #9
* Code has been re-platformed to a .NET6 project. Playwright requires a .csproj file.
* Initial re-factoring
* Build process now works fully on MacOS, Linux, and (of course) Windows.
* Added a code dependency on [svg-net](https://github.com/svg-net/SVG) for setting SVG height, width, viewbox, and aspect ratio. This could be eliminated in the future.
* Updated build documentation

### CODE REVIEW

**Comment by travisnielsen** [comment]
@Potherca just curious to know about any plans to review / merge this PR and the two that follow it. Are we looking for more reviewers?

**Comment by Potherca** [comment]
One reviewer is fine for now. At this point it is waiting for me or @RicardoNiepel to review the MRs.

Sadly my planning is currently dominated by homeschooling my children, as we are stuck in self-quarantine because one of my daughters has COVID. I'm hoping to find some time during the week, but it could be at a standstill for another week. 😞

**Comment by travisnielsen** [comment]
OK - no worries. Best wishes on your daughter's recovery!

**Comment by travisnielsen** [comment]
Hi @Potherca - Any plans to review these PRs?

File: scripts/Program.cs
Comment (Potherca) [inline scripts/Program.cs:17]: Missing newline, please add.

File: scripts/lib/IImageManager.cs
Comment (Potherca) [inline scripts/lib/IImageManager.cs:8]: Missing newline.

File: scripts/lib/Util.cs
Comment (Potherca) [inline scripts/lib/Util.cs:15]: Missing newline.

**Review by Potherca — APPROVED** [review]
I finally found some time to review this.

Everything looks alright. Some files are missing newline at end-of-file. I'll add those.

I'll run the code locally (I'm on Linux), after that, this can be merged.

**Comment by Potherca** [comment]
Running locally seems to work. Getting a timeout for several sprites.

@travisnielsen We might have to look into increasing the timeout setting.

**Comment by travisnielsen** [comment]
Thanks for the review @Potherca ! I think I have addressed all the items from your testing: 

- [x] Removed reference to appsettings.example
- [x] Addressed `Missing [..] color or mono` errors. That was due to case sensitivity in Linux (good catch!). There is still one error for `Azure Data Lake Analytics` but that's legit and the icon should be removed from `config.yaml` in a future pull request.
- [x] Fixed timeouts. Issue was caused by invalid `xmlns` attributes in the svg element. This prevents the SVG from being visible in Playwright (or any browser for that matter). Added error handling to clearly state the issue if it happens with future icons.
- [x] Removed instructions for `libgdiplus` setup on Linux. It's no longer required because I removed the dependency on the `svg-net` library.

**Comment by Potherca** [comment]
I've updated my branch locally and run the code again.Everything seems to work just fine! :+1: 

There are some errors and warnings, but those are all application owned.

<details><summary>All errors and warnings</summary>

- Error: `Missing Azure Data Lake Analytics color or mono`

- Fail: `Invalid svg namespace attribute.Confirm file has xmlns='http://www.w3.org/2000/svg' in the svg element.`
  - source/manual/Azure-Batch-AI_COLOR.svg. 
  - source/manual/Azure-SignalR-Service_COLOR.svg.

- Warning: `File found in multiple locations`
  - API Management Services
    - source/official/integration/10042-icon-service-API-Management-Services.svg
    - source/official/app services/10042-icon-service-API-Management-Services.svg
  - Application Insights
    - source/official/devops/00012-icon-service-Application-Insights.svg
    - source/official/management + governance/00012-icon-service-Application-Insights.svg
  - App Services
    - source/official/containers/10035-icon-service-App-Services.svg
    - source/official/app services/10035-icon-service-App-Services.svg
    - source/official/compute/10035-icon-service-App-Services.svg
  - Azure Synapse Analytics
    - source/official/databases/00606-icon-service-Azure-Synapse-Analytics.svg
    - source/official/analytics/00606-icon-service-Azure-Synapse-Analytics.svg
  - Batch Accounts
    - source/official/containers/10031-icon-service-Batch-Accounts.svg
    - source/official/compute/10031-icon-service-Batch-Accounts.svg
  - CDN Profiles
    - source/official/app services/00056-icon-service-CDN-Profiles.svg
    - source/official/networking/00056-icon-service-CDN-Profiles.svg
  - CDN Profiles
    - source/official/app services/00056-icon-service-CDN-Profiles.svg
    - source/official/networking/00056-icon-service-CDN-Profiles.svg
  - Container Instances
    - source/official/containers/10104-icon-service-Container-Instances.svg
    - source/official/compute/10104-icon-service-Container-Instances.svg
  - Cost Management And Billing
    - source/official/migrate/00004-icon-service-Cost-Management-and-Billing.svg
    - source/official/management + governance/00004-icon-service-Cost-Management-and-Billing.svg
    - source/official/general/00004-icon-service-Cost-Management-and-Billing.svg
  - Data Box Edge
    - source/official/storage/10095-icon-service-Data-Box-Edge.svg
    - source/official/migrate/10095-icon-service-Data-Box-Edge.svg
  - Data Box
    - source/official/storage/10094-icon-service-Data-Box.svg
    - source/official/migrate/10094-icon-service-Data-Box.svg
  - Event Hubs
    - source/official/iot/00039-icon-service-Event-Hubs.svg
    - source/official/analytics/00039-icon-service-Event-Hubs.svg
  - Function Apps
    - source/official/iot/10029-icon-service-Function-Apps.svg
    - source/official/compute/10029-icon-service-Function-Apps.svg
  - Instance Pools
    - source/official/databases/10139-icon-service-Instance-Pools.svg
    - source/official/other/10139-icon-service-Instance-Pools.svg
  - Kubernetes Services
    - source/official/containers/10023-icon-service-Kubernetes-Services.svg
    - source/official/compute/10023-icon-service-Kubernetes-Services.svg
  - Log Analytics Workspaces
    - source/official/management + governance/00009-icon-service-Log-Analytics-Workspaces.svg
    - source/official/analytics/00009-icon-service-Log-Analytics-Workspaces.svg
  - Logic Apps
    - source/official/iot/10201-icon-service-Logic-Apps.svg
    - source/official/internet of things/02631-icon-service-Logic-Apps.svg
    - source/official/integration/10201-icon-service-Logic-Apps.svg
  - Notification Hubs
    - source/official/iot/10045-icon-service-Notification-Hubs.svg
    - source/official/app services/10045-icon-service-Notification-Hubs.svg
  - Notification Hubs
    - source/official/iot/10045-icon-service-Notification-Hubs.svg
    - source/official/app services/10045-icon-service-Notification-Hubs.svg
  - Recovery Services Vaults
    - source/official/storage/00017-icon-service-Recovery-Services-Vaults.svg
    - source/official/migrate/00017-icon-service-Recovery-Services-Vaults.svg
    - source/official/management + governance/00017-icon-service-Recovery-Services-Vaults.svg
  - Service Fabric Clusters
    - source/official/containers/10036-icon-service-Service-Fabric-Clusters.svg
    - source/official/compute/10036-icon-service-Service-Fabric-Clusters.svg
  - SQL Data Warehouses
    - source/official/databases/00036-icon-service-SQL-Data-Warehouses.svg
    - source/official/integration/00036-icon-service-SQL-Data-Warehouses.svg
  - SQL Data Warehouses
    - source/official/databases/00036-icon-service-SQL-Data-Warehouses.svg
    - source/official/integration/00036-icon-service-SQL-Data-Warehouses.svg
  - Stream Analytics Jobs
    - source/official/iot/00042-icon-service-Stream-Analytics-Jobs.svg
    - source/official/analytics/00042-icon-service-Stream-Analytics-Jobs.svg


</details>

**Review by Potherca — APPROVED** [review]

**Comment by Potherca** [comment]
@travisnielsen I am all set to merge this. Is there anything that needs to be done before I do so?

After the merge https://github.com/plantuml-stdlib/Azure-PlantUML/pull/36 and https://github.com/plantuml-stdlib/Azure-PlantUML/pull/37/ can be updated / rebased / rerun / etc. and then be merged as well.

Thank you again for all your effort and patience to make this happen!

**Comment by travisnielsen** [comment]
> @travisnielsen I am all set to merge this. Is there anything that needs to be done before I do so?

Nope. We should be good to proceed with the merge!

> After the merge #36 and #37 can be updated / rebased / rerun / etc. and then be merged as well.

Yep. I'll rebase the other two branches after the merge to main. Those should go much more quickly since they don't involve code changes.

**Pull request merged**
